app/styles.module.css

```diff
@@ -5,12 +5,181 @@
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  margin: 0;
-  padding: 0;
+}
+.header {
+  display: flex;
+  flex-direction: column;
+  width: 342px;
+  height: 88px;
+  background: #fafafa;
+  display: flex; /* Makes it a flex container */
+  align-items: center; /* Vertically center items */
+  justify-content: space-between; /* Left, center, and right sections spread out */ /* Or whatever height you prefer */
+  padding: 0 16px;
+  border-bottom: 1px solid #c4c4c4;
 }
 
 .content {
-  max-width: 350px;
+  width: 375px;
+  height: 812px;
   height: 100%;
   border: 1px solid black;
+  font-style: poppins;
+}
+.profile {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-left: 15px;
+  margin-right: 15px;
+  margin-top: 15px;
+}
+
+.picName {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.name {
+  /* display:flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-start; */
+  font-weight: 500;
+  margin-left: 5px;
+  font-size: 85%;
+  align-self: center;
+}
+
+.avatar {
+  width: 22px;
+  height: 22px;
+  background-color: #ccc;
+  border-radius: 50%;
+  margin-right: 8px;
+}
+
+.date {
+  color: #888;
+  font-size: 11px;
+  align-self: last baseline;
+}
+
+.words {
+  margin: 15px;
+  font-size: 13px;
+}
+
+.pic {
```

Comment:
Use a <img> (or <Image> from 'next/image') to render images, instead of using CSS.